CMakeLists.txt

```diff
@@ -1,19 +1,16 @@
 # Copyright: (C) 2016 iCub Facility - Fondazione Istituto Italiano di Tecnologia
-# Authors: Francesco Romano <francesco.romano@iit.it>
+# Authors: Luca Tagliapietra <luca.tagliapietra@iit.it>
+#          Francesco Romano <francesco.romano@iit.it>
 # CopyPolicy: Released under the terms of the GNU LGPL v2.0+
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
```

Comment:
> The FATAL_ERROR option is accepted but ignored by CMake 2.6 and higher. 

https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html

Reply:
Sorry @traversaro, I've been too fast in merging. It was a copy-paste from some other CMakeList in the robotology I think. I'll remove it in the next commit then.